package.json

```diff
@@ -11,7 +11,7 @@
     "cypress:open": "cypress open",
     "cypress:run": "cypress run --spec 'cypress/integration/*.spec.js'",
     "cypress:coverage": "nyc npm run cypress:run",
-    "coverage:report": "nyc report --check-coverage --statements 50",
+    "coverage:report": "jest --coverage --coverageThreshold='{\"global\":{\"statements\":\"50\"}}'",
```

Comment:
Why are we only using `jest` coverage?

Reply:
Because sometimes nyc is giving empty coverage reports.

Reply:
https://github.com/cypress-io/code-coverage . This is the link I am using for cypress coverage but it is giving me empty results.

Reply:
I would recommend raising a separate PR for this coverage change, so that we can create an issue to fix this later.

Reply:
So, for now what should we go with in this PR. And what should a separate PR have.

Reply:
"coverage:report": "nyc report --check-coverage --statements 50", if we run this, it will sometimes fail our prod because there will be no coverage, which we don't want at this time.

Reply:
Let's create a separate PR for the `packag.json` change. We can merge this PR after the same.